src/apps/SocketsRegistry/SocketsRegistryList.js

```diff
@@ -130,58 +116,81 @@ const SocketsRegistryList = React.createClass({
     );
   },
 
-  render() {
-    const {
-      isLoading,
-      changeListView
-    } = this.props;
+  renderNotification() {
     const items = this.getFilteredData();
-    const styles = this.getStyles();
-    const socketImageDir = '/img/socket-assemble.svg';
-    const pluralizedResults = pluralize('result', items.length);
-    const containsItems = (!items || !items.length) && !isLoading;
+    const count = items.length;
 
-    if (containsItems && changeListView) {
+    if (!count) {
       return (
-        <div style={styles.noResultsState}>
-          <div style={styles.noResultsTitle}>
-            No Sockets found
-          </div>
-          <div style={styles.noResultsText}>
-            {`Sorry, but nothing matched your search criteria.
-            Please try again with some different keywords.`}
-          </div>
-        </div>
+        <Notification
+          hasCloseButtonVisible={false}
+          type="error"
+          className="vm-3-b"
+        >
+          No Sockets found. Sorry, please try again with some different keywords.
+        </Notification>
       );
     }
-    if (containsItems && !changeListView) {
+
+    const pluralizedResultText = pluralize('result', count);
+
+    return (
+      <Notification
+        hasCloseButtonVisible={false}
+        className="vm-3-b"
+      >
+        {`Found ${count} ${pluralizedResultText}.`}
+      </Notification>
+    );
+  },
+
+  renderContent() {
+    const items = this.getFilteredData();
+    const { isLoading, filter, filterBySyncano, term } = this.props;
+
+    if (!term || isLoading) {
       return (
         <RegistryEmptyView
           title="Supercharge your project with Sockets from the community"
-          description={`Think of Sockets Registry a package manager for Syncano Sockets. You can search for
-            Sockets created by community and add them to your projects.`}
-          src={socketImageDir}
+          description={`Think of Sockets Registry a package manager for Syncano Sockets. You can search for Sockets
```

Comment:
Think of Sockets Registry as a package manager     - sorry probably my previous mistake;p